tests/integration/components/o-s-s/array-input-test.js

```diff
@@ -69,27 +69,28 @@ module('Integration | Component | OSS::ArrayInput', function (hooks) {
     });
 
     test('it is triggered on keyword addition', async function (assert) {
-      await render(hbs`<OSS::ArrayInput @validator={{this.validator}}/>`);
+      await render(hbs`<OSS::ArrayInput @validator={{this.validator}} />`);
       await fillInputAndValidate('hashtag');
       assert.ok(validator.calledWith('hashtag'));
     });
 
     test('WHEN the validator is truthy, the value is added as a tag', async function (assert) {
       validator.returns(true);
-      await render(hbs`<OSS::ArrayInput @validator={{this.validator}}/>`);
+      await render(hbs`<OSS::ArrayInput @validator={{this.validator}} />`);
       await fillInputAndValidate('hashtag');
       assert.ok(validator.calledWith('hashtag'));
       assert.ok(validator.returned(true));
-      assert.dom('.input-array-tag').hasText('hashtag');
+      assert.dom('.upf-chip').exists();
+      assert.dom('.array-input-container').hasText('hashtag');
```

Comment:
In your tests, you change `.input-array-tag` into `upf-chip` or `.array-input-container`, there is a reason for this?

Reply:
.input-array-tag isn't use anymore so i delete it and i've one test to update in publisher admin